doc/contributor/cexts.md

```diff
@@ -5,20 +5,28 @@
 TruffleRuby runs C extension using Sulong. You should build Sulong from source.
 Clone from https://github.com/graalvm/sulong
 
-You need LLVM installed - version 3.8 or above.
+You need LLVM installed - version 3.8 or above we think but we haven't tested
+many versions.
 
 We need the `opt` command, so you can't just use what is installed by Xcode if
 you are on macOS. We would recommend that you install LLVM via Homebrew (`brew
 install llvm`) and then manually set your path in shells where you are building
 Sulong or TruffleRuby (`export PATH="/usr/local/opt/llvm/bin:$PATH"`).
 
-You can now build the C extension support. Building the OpenSSL C extension is
-incomplete, so most people probably want to disable that with `--no-openssl`.
+You also need OpenSSL - version 1.0.1 or above we think but we haven't tested
+many versions. The system version of OpenSSL on macOS is too old, so you should
+install a new version, again via Homebrew (`brew install openssl`). If you are
+on macOS, or otherwise even if you just want to use a different version of
+OpenSSL you then need to set `OPENSSL_PREFIX` to tell us where to find it (for
+example `export OPENSSL_PREFIX=/usr/local/Cellar/openssl/1.0.2g`
```

Comment:
Does `/usr/local/opt/openssl` points to the same location?
If so it would avoid being dependent on the current latest version in Homebrew.